login-workflow/docs/language-support.md

```diff
@@ -0,0 +1,78 @@
+# Language Support
+
+This package supports translations to different langauges using [i18next](https://www.i18next.com/) / [react-i18next](https://github.com/i18next/react-i18next). The workflow screens are currently available in:
```

Comment:
typo here - languages

Reply:
Looks like I need to install my spellchecker plugin on the new mac.